src/containers/Tenant/Diagnostics/Diagnostics.tsx

```diff
@@ -76,15 +78,10 @@ function Diagnostics(props: DiagnosticsProps) {
             pages = TABLE_PAGES;
         }
         return pages;
-    }, [props.type]);
+    }, [props.type, isDatabase]);
```

Comment:
I suppose this fix is primarily for dev environment, where the root is considered to be a database, though having the type of a directory. But with the present logic this change is only reasonable, since it assumes this kind of situation, when `isDatabase` conflicts with `props.type`.